src/test/Assets/TestProjects/LightupClient/LightupClient.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>LightupClient</AssemblyName>
```

Comment:
Like before, AssemblyName and PackageId can be removed.  Repeated for the rest of the TestAssets projects.